### PR TITLE
feat: Update ExplorerDetails to use secondsToTimeString for TTL forma…

### LINKED ISSUE #418: More accurate formatting TTL function
The current Swarm blob data expiry indicator in the header is not accurate.

image

Here it is displayed "a month" although we know it is "5 weeks" at the moment of writing this from the swarm dashboard.

Migrate from current `formatTtl()` to [secondsToTimeString](https://github.com/ethersphere/bee-dashboard/blob/master/src/utils/index.ts#L175) from bee-dashboard

### PR DESCRIPTION
**Pull request by luis-herasme**
### Description
More accurate formatting TTL function

### Related Issue
Closes #418

### CODE REVIEW

File: apps/web/src/utils/date.ts [hunk 1]
```diff
@@ -32,8 +32,35 @@ export function formatDate(
   return date;
 }
 
-export function formatTtl(ttl: number) {
```
Comment (PJColombo) [inline apps/web/src/utils/date.ts:35]: I feel like `formatTtl` was more readable and self-explanatory

File: .changeset/few-geese-ring.md [hunk 2]
```diff
@@ -0,0 +1,5 @@
+---
+"@blobscan/web": minor
+---
+
+refactor: Updated ExplorerDetails to use secondsToTimeString for TTL formatting
```
Comment (PJColombo) [inline .changeset/few-geese-ring.md:5]:
```suggestion
Enhanced Swarm data expiry formatting
```

**Review by PJColombo — CHANGES_REQUESTED** [review]
Nice job

Left a few comments

File: apps/web/src/utils/date.ts (hunk 1 above)
Comment (luis-herasme) [reply apps/web/src/utils/date.ts:35]: Done [fb307b4](https://github.com/Blobscan/blobscan/pull/503/commits/fb307b4011a4ef098de11c602ad75819b6c11ca8)

**Review by xFJA — COMMENTED** [review]
Good job! I left a discussion regarding adding UTs

File: apps/web/src/utils/date.ts [hunk 3]
```diff
@@ -32,8 +32,35 @@ export function formatDate(
...
-  return dayjs().to(dayjs().add(ttl, "second"), true);
+export function formatTtl(seconds: number) {
```
Comment (xFJA) [inline apps/web/src/utils/date.ts:35]: I wonder if we should add UTs here

Comment (PabloCastellano) [inline apps/web/src/utils/date.ts:35]: A comment giving credits to bee-dashboard and linking to https://github.com/ethersphere/bee-dashboard/blob/master/src/utils/index.ts#L175 would be nice here.

Comment (luis-herasme) [reply apps/web/src/utils/date.ts:36]: Done [6d85b8a](https://github.com/Blobscan/blobscan/pull/503/commits/6d85b8a6cb16c56c4303a1c36ebcaa6e72acc4e9)

**Review by PabloCastellano — APPROVED** [review]
Good! Last thing, could you add unit tests like Fran suggested?  I'm approving this in advance.

**Comment by PJColombo** [comment]
> Good! Last thing, could you add unit tests like Fran suggested? I'm approving this in advance.

Let’s focus on setting up tests for the components rather than the utility functions. We can discuss the best frameworks and approaches for this since we don’t have anything in place yet.

**Pull request merged**